fv/bpf_test.go

```diff
@@ -617,6 +617,160 @@ func describeBPFTests(opts ...bpfTestOpt) bool {
 					cc.CheckConnectivity()
 				})
 
+				It("should not be able to spoof workloads IP by another workload", func() {
```

Comment:
If useful, you can add a config param to felix prefixed with `Debug...` to disable the RPF check in  order to cross-check that the only thing dropping the packet is the new check.  We have a few of those for testing normally-unhittable cases, such as `DebugSimulateDataplaneHangAfter`

Reply:
good point, probably does not matter here, I escaped those manually